### PR TITLE
Log the process

### PR DESCRIPTION
**Pull request by viktorprogger**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #23

### CODE REVIEW

**Comment by viktorprogger** [comment]
[Queue](https://trello.com/c/QTPMAx67/6-queue)

**Comment by samdark** [comment]
https://travis-ci.com/github/yiisoft/yii-queue/builds/154603349

File: src/Queue.php [hunk 1]
```diff
@@ -75,12 +80,20 @@ public function jobRetry(JobFailure $event): void
...
     {
+        $this->logger->debug('Preparing to push job "{job}".', ['job' => get_class($job)]);
         $event = new BeforePush($this, $job);
         $this->eventDispatcher->dispatch($event);
 
         if ($this->driver->canPush($job)) {
             $message = $this->driver->push($job);
+            $this->logger->debug('Successfully pushed job "{job}" to the queue.', ['job' => get_class($job)]);
         } else {
+            $context = [
```
Comment (romkatsu) [inline src/Queue.php]: extra variable

```php
$this->logger->error(
    'Job "{job}" is not supported by driver "{driver}"',
    [
        'job' => get_class($job),
        'driver' => get_class($this->driver),
    ]
);
```

**Comment by samdark** [comment]
Thanks 👍

**Pull request merged**